discovery_engine_core/web-api/src/storage.rs

```diff
@@ -163,11 +177,19 @@ impl UserState {
 
         let deletion = sqlx::query("DELETE FROM center_of_interest;")
             .execute(&mut tx)
-            .await?;
+            .await?
+            .rows_affected()
+            > 0;
+        let deletion = sqlx::query("DELETE FROM interaction;")
+            .execute(&mut tx)
+            .await?
+            .rows_affected()
+            > 0
+            || deletion;
 
         tx.commit().await?;
 
-        Ok(deletion.rows_affected() > 0)
+        Ok(deletion)
```

Comment:
this will be removed by #621 so maybe don't add anything here to avoid conflicts